src/dump.ts

```diff
@@ -147,57 +150,166 @@ export interface ApiDump {
-            vscode.window.showErrorMessage("Error downloading API dump", err.toString())
-        }),
-    )
+function createDescribedClasses(rmd: RMDump, classes: ApiClass[]): ApiClass[] {
+    const describedClasses: ApiClass[] = []
 
     for (const classEntry of classes) {
-      const entry = rmd.roblox.Item.find(
-        (i: any) => i.$.class === "ReflectionMetadataClasses",
-      ).Item.find((i: any) =>
-        i.Properties[0].string.find(
-          (p: any) => p.$.name === "Name" && p._ === classEntry.Name,
+        const entry = rmd.roblox.Item.find(
+            (i: RMDParentItem) => i.$.class === "ReflectionMetadataClasses",
+        )?.Item.find((i: RMDItem) =>
+            i.Properties[0].string.find(
+                (p: RMDStringProperty) => p.$.name === "Name" && p._ === classEntry.Name,
+            ),
         )
-      )
-
-      const summary = entry?.Properties[0].string.find(
-        (s: any) => s.$.name === "summary",
-      )?._
-
-      if (entry && entry.Item) {
-        const items = Object.fromEntries(
-          entry.Item.flatMap((i: any) => i.Item)
-            .filter((i: any) => i && i.Properties !== undefined)
-            .map((i: any) => [
-              i.Properties[0].string.find((s: any) => s.$.name === "Name")?._,
-              i.Properties[0].string.find((s: any) => s.$.name === "summary")?._,
-            ])
-            .filter((entry: any) => entry.length === 2),
+
+        const summary = entry?.Properties[0].string.find(
+            (s: RMDStringProperty) => s.$.name === "summary",
+        )?._
+
+        const describedMembers: ApiMember[] = []
+
+        if (entry !== undefined && entry.Item !== undefined) {
+            const items: { [name: string]: string } = Object.fromEntries(
+                entry.Item.flatMap((i: RMDItem) => i.Item)
+                    .filter<RMDItem>(
+                        (i: RMDItem | undefined): i is RMDItem => i !== undefined && i.Properties !== undefined)
+                    .map((i: RMDItem) => [
+                        i.Properties[0].string.find((s: RMDStringProperty) => s.$.name === "Name")?._,
+                        i.Properties[0].string.find((s: RMDStringProperty) => s.$.name === "summary")?._,
+                    ])
+                    .filter((output: Array<string | undefined>) => output.length === 2),
+            )
+
+            for (const member of classEntry.Members) {
+                const describedMember: ApiMember = Object.assign({}, member)
+
+                if (items[member.Name] !== undefined) {
+                    describedMember.Description = items[member.Name]
+                }
+
+                describedMembers.push(describedMember)
+            }
+        }
+
+        const describedClass: ApiClass = Object.assign({}, classEntry,
+            {
+                Description: summary,
+                Members: describedMembers,
+            })
+        describedClasses.push(describedClass)
```

Comment:
This should be equivalent to:

```suggestion
        describedClasses.push({
                Description: summary,
                Members: describedMembers,
                ...classEntry
        })
```

Reply:
Yep, I'll change that (except I believe the spread has to go before the overwritten properties)

Reply:
You're right, yeah.